code/modules/religion/religion_sects.dm

```diff
@@ -277,8 +305,33 @@
 	rites_list = list(/datum/religion_rites/plantconversion, /datum/religion_rites/photogeist)
 	altar_icon_state = "convertaltar-green"
 
-//plant sect bibles don't heal or do anything special apart from the standard holy water blessings
-/datum/religion_sect/plant/sect_bless(mob/living/blessed, mob/living/user)
+//plant sect bibles will only heal plant-like things
+/datum/religion_sect/plant/sect_bless(mob/living/L, mob/living/user)
```

Comment:
this heals every damaged limb by 10, is this intended? or just one limb?

Reply:
Either works tbh but it was supposed to be one limb

Reply:
> 

fix

Reply:
ive changed my mind its now intended and no longer requires me to put more effort into this PR